tests/Integration/WorkloadManagementSystem/Test_GenerateAndExecutePilotWrapper.py

```diff
@@ -74,12 +74,9 @@
 from PilotWrapper import pilotWrapperScript  # pylint: disable=import-error
 
 diracInstall = os.path.join(os.getcwd(), 'dirac-install.py')
-with open(diracInstall, "r") as fd:
+with open(diracInstall, "rb") as fd:
   diracInstall = fd.read()
-if sys.version_info >= (3,):
-  diracInstallEncoded = base64.b64encode(bz2.compress(diracInstall.encode(), 9))
-else:
-  diracInstallEncoded = base64.b64encode(bz2.compress(diracInstall, 9))
+diracInstallEncoded = base64.b64encode(bz2.compress(diracInstall, 9)).decode()
```

Comment:
so, this variable called `somethingEncoded` is in fact a decoded bytes into string of a base 64 encoded bytes... and who said py3 was easier and more logical ? :D

Reply:
Base64 is [defined to be encoded in ASCII](https://en.wikipedia.org/wiki/Base64). The only way to achieve that for all possible system locales is to return a `bytes` object so technically the `.decode()` shouldn't be there but then I'd have to start fixing `pilotWrapperScript`.